jenkins.yaml

```diff
@@ -22,26 +28,27 @@ environment:
       - git
       - glibc-locale-en
       - maven
-      - openjdk-17
-      - openjdk-17-default-jvm
+      - openjdk-11
```

Comment:
Intentional change - mirrors upstream, compiles with JDK11 but supports runtime of JDK11, JDK17, or JDK21.